instrumentation/grape/README.md

```diff
@@ -1,6 +1,15 @@
 # OpenTelemetry Grape Instrumentation
 
-Todo: Add a description.
+The Grape instrumentation is a community-maintained instrumentation for [Grape](https://github.com/ruby-grape/grape), a REST-like API framework for Ruby.
+
+It relies mostly on the Grape built-in support for `ActiveSupport::Notifications` (more info [here](https://github.com/ruby-grape/grape#active-support-instrumentation)).
+
+It currently supports the following events:
+
+- `endpoint_run.grape`
+- `endpoint_render.grape`
+- `endpoint_run_filters.grape`
+- `format_response.grape`
```

Comment:
>if format_response blows up the http.status_code for endpoint_run will still be 200 😢

true but depending on the defined middleware this response can be an error response and thus tracked by an upstream span (I am thinking rack middleware generated span)

PS: not disagreeing adding to the information presented here.

Reply:
Thank you for the clarification 🙇‍♀️  And yes, I think it will be ok 😄
But I just wanted to point it out (and I plan to leave this comment in the upstream PR so the maintainers are fully aware).